rubi/scripts.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+
+def test(args=sys.argv):
```

Comment:
One thing I want to call your attention to @denverbaumgartner is adding flags when testing. Individual tests can be called with the `-k <test_name>` flag but I have to add in logic to do that when running `poetry run test`. 

This is a way to do it but could break if the user does not specify the `-k` first and then `test_name`. I've made it explicit in the README but just wanted to get your thoughts on this

Another way of doing it can be creating a whole new function `def test_specific` for example and just pass in the name of the test and handling the -k in the backend

Reply:
im ok with just requiring a flag for right now, thanks for flagging this ;)

Reply:
Don't think this is an issue either